### PR TITLE
Dashboard and various improvements

### PR DESCRIPTION
**Pull request by FredrikAugust**
Knowing that we're working on a tight schedule, combined with an excess of coffee, I got carried away, and fixed some things that were lagging behind. Sorry about that, I know it violates the "One PR - One Change" policy.

Things that are implemented:

- On dashboard; chart with previous 3 and coming 9 months of balance for all companies logged in to
- Showing current balance as well as change from previous month on dashboard
- Projections now support recurring transactions and changing company (will update accordingly)
- You now have access to all the features of a company
	- Create new ones
	- Edit name and org_nr
	- Leave the company
	- Add other to company
	- Change roles of others in company
	- Kick others from company
- Normalize styling
- Implement error handling from backend
- Fix a bug where you would be logged out when entering page even though refresh token was still valid

Changes made by @NicolajNiels1 which were incorporated into this PR:

- Remove balances page, as it is no longer needed
- Add projections page

### CODE REVIEW

**Comment by FredrikAugust** [comment]
This makes #157 redundant, as the changes are incorporated into this one, including a small graphical overhaul, as well as support for rec. tx.

**Comment by FredrikAugust** [comment]
Appears to be a bug in the calculation of growth in percentage. Fix incoming.

File: frontend/src/components/atoms/Routes.tsx [hunk 1]
```diff
@@ -17,6 +18,18 @@ import Page from '../templates/Page';
 const Routes: React.FC<RouteComponentProps<{}>> = props => {
   const user = useAuthState();
 
+  /**
```
Comment (flyrev) [inline frontend/src/components/atoms/Routes.tsx]: This bug fix should be its own pull request.

Could you open a separate issue for this bug, and then create a new pull request with this fix?

File: frontend/src/styling/global.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export const GlobalStyle = createGlobalStyle`
 
   /* Typography */
 
-  h1, h2, h3, h4, h5, h6, p, a, span, li, strong, label, input, textarea, option, select, button {
+  * {
```
Comment (flyrev) [inline frontend/src/styling/global.ts]: Is there a specific reason you use `*` instead of `body`?

Also, this change should happen in a separate pull request. Could you create a new pull request for this which only has this change? You can call it "Set default font for the application" or something similar.

Comment (FredrikAugust) [reply frontend/src/styling/global.ts]: Yes. This is to reset the padding/margin of every html element. The reasoning behind this is to force the developer to implement their own styling for all elements, and not rely on the user-agent stylesheets, who have a tendency to differ slightly, leading to inconsistencies in padding/margin.

Comment (FredrikAugust) [reply frontend/src/styling/global.ts]: Changing this to body would require us to manually remove the padding/margin on all other elements touched by the user-agent stylesheet.

File: frontend/src/components/atoms/Routes.tsx (hunk 1 above)
Comment (FredrikAugust) [reply frontend/src/components/atoms/Routes.tsx]: Done #169.
